cmd/operator/internal/controller/sinkfilter_controller_test.go

```diff
@@ -0,0 +1,283 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package controller
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	sourcesv1 "knative.dev/eventing/pkg/apis/sources/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	kubearchivev1 "github.com/kubearchive/kubearchive/cmd/operator/api/v1"
+	"github.com/kubearchive/kubearchive/pkg/constants"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("SinkFilterController", func() {
```

Comment:
I'm wondering, until now we were doing more standard tests without the ginkgo library, why did you introduce it? Is the recommended way of testing controllers? And if so, should we consider migrating the tests of the rest of kubearchive to ginkgo usage?

Reply:
ok!